exercise-1/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #3da2e0;
+}
+div {
+  background-color: white;
+  height: 620px;
+  width: 620px;
+  border-radius: 10px;
+  margin: 44px 0px 0px 360px;
+}
+.name {
+  padding: 40px 118px 10px 13px;
+  border-width: 0 0 1px;
+}
+.password {
+  padding: 40px 120px 10px 13px;
+  border-width: 0 0 1px;
+}
+.login {
+  padding: 2px 1px 0px 0px;
+  margin: 40px 0px 0px -4px;
+  background-color: #ec3d67;
+  width: 313px;
+  border-radius: 34px;
+  height: 46px;
+  border: 0px;
+  color: white;
+  cursor: pointer;
+}
+.img1 {
```

Comment:
img1,2,3 are not good and expressive class names. I can't infer from CSS that img2 is the username icon. Please fix this asap. I'll approve this PR then.